### ISSUE TITLE
[Question] server zip contents

### ISSUE THREAD

**Issue by Master-Jynx** [Sunekaer/stoneBlock#755]
my apologies, for asking questions here, but you appear to have disabled comments on curse forge.
What server is included in the server pack?
what plugins  are included for Permissions? for Admin Tools? for fixing Giefing ? World Edit Support?
For world backups ? for lag reduction , such as Ground Cleaner?

**Comment by Master-Jynx** [comment]
I asked abut the server pack not ssp

**Comment by Sunekaer** [comment]
yes it is, the pack is forge, so it the serverfiles is forge server files with the same mods and config as the pack has

**Comment by Master-Jynx** [comment]
Thank you and merry christmas
